src/background/user-configuration-controller.ts

```diff
@@ -25,10 +24,9 @@ export class UserConfigurationController {
         const payload: SetTelemetryStatePayload = {
             enableTelemetry,
         };
-        const message: Message = {
+        const message: InterpreterMessage = {
             messageType: Messages.UserConfig.SetTelemetryConfig,
             payload: payload,
-            tabId: null,
```

Comment:
guess we need tabId here?

Reply:
We removed tabId from here as was getting yarn null check error
![NullCheck](https://github.com/microsoft/accessibility-insights-web/assets/150342469/4bb1be55-fa47-4f6f-ab52-c0fd822bb903)